app/src/androidTest/java/com/github/sdpcoachme/EditProfileActivityTest.kt

```diff
@@ -133,4 +140,106 @@ class EditProfileActivityTest {
             }
         }
     }
+
+    @Test
+    fun requestForExistingEmailDisplaysCorrectInfoInUserFields() {
+        val editProfileIntent = Intent(ApplicationProvider.getApplicationContext(), EditProfileActivity::class.java)
+        val email = "some@mail.com"
+        val db = (InstrumentationRegistry.getInstrumentation().targetContext.applicationContext as CoachMeApplication).database
+        db.addUser(
+                UserInfo(
+                "first",
+                "last",
+                email,
+                "012345",
+                "Some Place",
+                false,
+                    listOf()
```

Comment:
Low priority feedback/suggestion: This test makes sense and is well-written overall, the indentation here for the new user is not very nice. Would be cool to have that formatting adapted to make the distinction between the constructor name and the arguments clearer.